src/actions/hover.rs

```diff
@@ -543,35 +543,36 @@ fn create_tooltip(
 /// ```
 /// # use std::path::Path;
 ///
-/// let base_path = Path::from(".rustup/toolchains/nightly-x86_64-pc-windows-msvc/lib/rustlib/src/rust/src/liballoc/string.rs");
-/// let tidy_path = skip_path_components(base_path.to_path_buf(), ".rustup", 8);
-/// assert_eq!("liballoc/string.rs", tidy_path);
+/// let base_path = Path::new(".rustup/toolchains/nightly-x86_64-pc-windows-msvc/lib/rustlib/src/rust/src/liballoc/string.rs");
+/// let tidy_path = skip_path_components(base_path, ".rustup", 7);
+/// assert_eq!(tidy_path, Some(PathBuf::from("liballoc/string.rs")));
 ///
-/// let base_path = Path::from(".cargo/registry/src/github.com-1ecc6299db9ec823/smallvec-0.6.2/lib.rs");
-/// let tidy_path = skip_path_components(base_path.to_path_buf(), ".cargo", 4);
-/// assert_eq!("smallvec-0.6.2/lib.rs", tidy_path);
+/// let base_path = Path::new("/home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/smallvec-0.6.2/lib.rs");
+/// let tidy_path = skip_path_components(base_path, "/home/user/.cargo", 3);
+/// assert_eq!(tidy_path, Some(PathBuf::from("smallvec-0.6.2/lib.rs")));
```

Comment:
I've always followed the `assert(expected, actual)` convention that junit uses, but I guess the assert macro doesn't make that distinction.

e.g. https://junit.org/junit4/javadoc/4.12/org/junit/Assert.html#assertEquals(long,%20long)

Reply:
I certainly didn't mean to force any changes! It's just that it read better for me due to symmetry with assigning above, so the variable is on the left side and checked expression is on the right (and the macro doesn't make any distinction, as you've said):
```rust
let var = compute_expr();
assert_eq!(var, val_of_expr);
```

Reply:
Tbf the general consensus is `assert_eq!(actual, expected)` you can see this in rust docs
```rust
// len docs
let mut v = HashSet::new();
assert_eq!(v.len(), 0);
v.insert(1);
assert_eq!(v.len(), 1);
```

I think most language/test apis use this way too, junit didn't but hamcrest & assertj, for example, do.